lib/elixir/lib/kernel.ex

```diff
@@ -2973,15 +2973,15 @@ defmodule Kernel do
 
       false when doc_attr? ->
         quote do
-          case Module.get_attribute(__MODULE__, unquote(name), unquote(line)) do
+          case Module.__get_attribute__(__MODULE__, unquote(name), unquote(line)) do
             {_, doc} -> doc
             other -> other
           end
         end
 
       false ->
         quote do
-          Module.get_attribute(__MODULE__, unquote(name), unquote(line))
+          Module.__get_attribute__(__MODULE__, unquote(name), unquote(line))
```

Comment:
I like this a lot. :heart: For consistency, can you please rename the private `put_attribute` to `__put_attribute__` too?

Reply:
I renamed it in next commit. Thanks !